src/components/FeatureViews/Accounts/ProfileHeading.js

```diff
@@ -233,6 +260,17 @@ class ProfileHeading extends Component {
                         {this.props.isAdmin && this.renderEditButton()}
                     </Grid>
                 </Grid>
+                <Grid>
+                    <Button onClick={(e) => {
+                                e.preventDefault();
+                                this.handleOpen(e);
+                            }}>
+                    asd
```

Comment:
Properly label this button as "Set OOO"

Reply:
Also add a tooltip saying "Set Out Of Office Event". Here's [the documentation for tooltips](https://v3.material-ui.com/demos/tooltips/)